app/views/events/_fields.slim

```diff
@@ -1,25 +1,25 @@
-.Form
-  .Form-label
-    = f.label :name_name
-  .Form-input
+.FormBodyLayout
+  .FormBodyLayout-label
+    .Label = f.label :event_name
```

Comment:
The .Label component could be applied to the actual label tag, instead of creating one more nested div